src/frontends/arcs/handle_connection_spec_test.cc

```diff
@@ -120,25 +118,20 @@ TEST_P(GetAccessPathTest, GetAccessPathTest) {
 }
 
 // Just some different strings that could be used as particle specs.
-std::string sample_particle_spec_names[] = {
-    "ParticleSpec",
-    "PS",
-    "P1",
-    "P2"
-};
+std::string sample_particle_spec_names[] = {"ParticleSpec", "PS", "P1", "P2"};
```

Comment:
Since we're touching the code, this should probably be turned into `static constexpr absl::string_view sample_particle_spec_names[]`.

Reply:
done.